go/cmd/vtadmin/main.go

```diff
@@ -163,6 +171,8 @@ func main() {
 
 	// rbac flags
 	rootCmd.Flags().StringVar(&rbacConfigPath, "rbac-config", "rbac.yaml", "")
+	rootCmd.Flags().BoolVar(&enableRbac, "rbac", false, "whether to enable rbac")
+	rootCmd.Flags().BoolVar(&disableRbac, "no-rbac", false, "whether to disable rbac")
```

Comment:
i'm not sure which is better; i'm curious what you think about this!

```go
rootCmd.Flags().StringVar(&rbacConfigPath, "rbac-config", "", "path to an RBAC config file. must be set if not pasing --no-rbac") // now defaulting to empty
rootCmd.Flags().BoolVar(&disableRBAC, "no-rbac", false, "whether to disable RBAC. must be set if not passing --rbac-config")
```

and then in the run func:

```go
if rbacConfigPath == "" {
	if !disableRBAC {
		// fail, must pass exactly one
	}

	rbacConfig = rbac.DefaultConfig()
} else if disableRBAC {
	// fail, must pass exactly one
} else {
	cfg, err := rbac.LoadConfig(rbacConfigPath)
	if err != nil {
		fatal(err)
	}

	rbacConfig = cfg
}
```

Reply:
I like removing the default for the rbac config. I guess the other difference between this and the other is that the current implemention requires explicitly passing an "enable rbac" flag, specified by the issue: https://github.com/vitessio/vitess/issues/9419. I personally prefer explicitly enabling rbac per the issue 😄